src/main/resources/init_data.sql

```diff
@@ -1,127 +0,0 @@
-INSERT INTO public.user_profile (id, first_name, last_name, email, username, password) VALUES ('8f8fc016-5bb2-4906-ad88-68932c438665', 'John', 'Smith', 'smith.j@sample.com', 'user1', '$2a$08$6PEeGWWVF1M8hU8Y8XCNfe/FAs4WGHXX/wB0znhC3O3TxYtWY/xxe');
-INSERT INTO public.user_profile (id, first_name, last_name, email, username, password) VALUES ('66c34e55-9dec-483f-8458-1fc50e802415', 'Asa', 'Edward', 'edward.a@sample.com', 'user2', '$2a$08$gAG7ItEHUDaigRlfi9xreeK3zdbYvOsqF6x0Ic8gZpyVgK6d3j9Ba');
```

Comment:
Didn't see a point in keeping `init_data.sql` anymore. I don't think people use it that much, but correct me of I'm wrong. I just think loading data through `Data.ts` is much easier to maintain